src/Compilers/CSharp/Portable/Emitter/Model/PEModuleBuilder.cs

```diff
@@ -1385,5 +1385,17 @@ protected override Cci.IMethodDefinition CreatePrivateImplementationDetailsStati
         {
             return new SynthesizedPrivateImplementationDetailsStaticConstructor(SourceModule, details, GetUntranslatedSpecialType(SpecialType.System_Void, syntaxOpt, diagnostics));
         }
+
+        internal virtual SynthesizedAttributeData SynthesizeEmbeddedAttribute()
+        {
+            // For modules, this attribute should be present. Only assemblies generate and embedd this type.
+            return Compilation.TrySynthesizeAttribute(WellKnownMember.Microsoft_CodeAnalysis_EmbeddedAttribute__ctor);
```

Comment:
It is useful in the case of building a module. The module should be able to look up referenced attributes (possibly in mscorlib) normally.

---
In reply to: [113569994](https://github.com/dotnet/roslyn/pull/18715#discussion_r113569994) [](ancestors = 113569994)